README.md

```diff
@@ -1,44 +1,26 @@
-[![Open in Codespaces](https://classroom.github.com/assets/launch-codespace-7f7980b617ed060a017424585567c406b6ee15c891e84e1186181d67ecf80aa0.svg)](https://classroom.github.com/open-in-codespaces?assignment_repo_id=11521483)
-# Sphinx Profile Page
+# Profile Website
 
-This is is a template for a resume/ personal portfolio, built with Sphinx! Based on [Chris Holdgraf's personal site](https://github.com/choldgraf/choldgraf.github.io)
+Welcome to my profile webiste, 
+the target audience might 18 to 25 year old people who want money 
 
-This template is designed for use in the URI CS TD SSP. 
+Since you're here, you might be: 
+  - looking for cool pictures 
+  - finding things to learn about me  
+  - looking for a new friend 
+  - job employers
+  -  anyone 
 
+   ## design 
```

Comment:
For your ## Design section, describe how you will get your message across using design. Your description should include details on how you will use visual hierarchy, grouping and similarity.